sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2100,6 +2281,26 @@ def hpat_pandas_series_prod_impl(self, axis=None, skipna=True, level=None, numer
 @overload_method(SeriesType, 'quantile')
 def hpat_pandas_series_quantile(self, q=0.5, interpolation='linear'):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.quantile
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/quantile.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the value at the given quantile
+       :name: ex_series_quantile
+
+    .. code-block:: console
+
+        > python ./series/quantile.py
+        2.5
+
+    Intel Scalable Dataframe Compiler Developer Guide
```

Comment:
Missing See Also. Pandas has at least references to Rolling.quantile and numpy.percentile

Reply:
Currently we don't have support for Rolling.quantile so I am not able to add a link